src/webapi/Service/AuthenticationConfigurationService.cs

```diff
@@ -0,0 +1,44 @@
+using EdFi.OdsApi.Sdk.Client;
+using EdFi.OdsApi.SdkClient;
+
+namespace eppeta.webapi.Service
+{
+    public class AuthenticationConfigurationService : IAuthenticationConfigurationService
```

Comment:
This class seems pretty good and you've done a nice job with dependency injection.

There is potential confusion here for future developers. There are two uses of authentication in this application: end-users logging in, and the service connecting to the Ed-Fi ODS/API. It may be useful to rename this class to help make it more clear that this serves the second purpose.